Jenkinsfile

```
@@ -294,8 +294,9 @@ void applyCustomizations() {
 void runTests() {
   sh '''
     export PIP_INDEX_URL=https://${ARTIFACTORY_CREDS_USR}:${ARTIFACTORY_CREDS_PSW}@${ARTIFACTORY_URL_DOWN##'https://'}/api/pypi/cloudant-sdks-pypi-virtual/simple
-    python3 -m tox -e py311-lint
-    python3 -m tox -e py311
+    pip install . && pip install -r requirements-dev.txt
+    pylint --rcfile=.pylintrc ibmcloudant test
+    py.test --reruns 3 -o junit_family=xunit2 --junitxml=junitreports/junit-pytest.xml --cov=ibmcloudant
```

We should use the `-m` version to make sure our working dir is on the search path for imports. Additionally my understanding is that xunit2 is the default nowadays and I hope we don't think we need to do reruns here
```suggestion
    python3 -m pytest --junitxml=junitreports/junit-pytest.xml --cov=ibmcloudant
```